mlflow/R/mlflow/.create-test-env.R

```diff
@@ -13,14 +13,3 @@ reticulate::conda_install("'h5py<3.0.0'", envname = mlflow:::mlflow_conda_env_na
 reticulate::conda_install(Sys.getenv("MLFLOW_HOME", "../../../../."), envname = mlflow:::mlflow_conda_env_name(), pip = TRUE)
 reticulate::conda_install("xgboost", envname = mlflow:::mlflow_conda_env_name())
 reticulate::conda_install(paste0("h2o==", packageVersion("h2o")), envname = mlflow:::mlflow_conda_env_name(), pip = TRUE)
-
-# The default timeout value (60 seconds) can be insufficient for `spark_install` to complete
-options(timeout=60 * 60)
-
-spark_version_file <- "../.spark-version"
-spark_version <- trimws(readChar(spark_version_file, file.info(spark_version_file)$size))
-
-# Install MLeap runtime and required dependencies
-sparklyr::spark_install(version = spark_version, verbose = TRUE)
-mleap::install_maven()
-mleap::install_mleap(version = "0.16.0")
```

Comment:
I this the reason these lines suddenly started failing today is probably because cache for R has expired.